.github/workflows/check-consistent-dependencies.yml

```
@@ -28,7 +28,7 @@ jobs:
           echo $'Paths touched in PR:\n'"$paths"
 
           # The ^"? is because git may quote weird file paths
-          matched="$(echo "$paths" | grep -P '^"?requirements/' || true)"
+          matched="$(echo "$paths" | grep -P '^"?((requirements/)|(scripts/.*?/requirements/))' || true)"
```

That should work, although you can drop some of the grouping parens:

```suggestion
          matched="$(echo "$paths" | grep -P '^"?(requirements/|scripts/.*?/requirements/)' || true)"
```

Or, to make it more future-proof we could just ask if there are any path segments at all that are `requirements` (since it's only a performance issue if this workflow triggers more broadly than necessary):

```suggestion
          matched="$(echo "$paths" | grep -P '(^|["/])requirements/' || true)"
```